test/TestServerProjects/lro/Generated/LRORetrysDelete202Retry200Operation.cs

```diff
@@ -26,17 +26,20 @@ protected LRORetrysDelete202Retry200Operation()
 
         internal LRORetrysDelete202Retry200Operation(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, Request request, Response response)
         {
-            _operation = new OperationInternals(clientDiagnostics, pipeline, request, response, OperationFinalStateVia.Location, "LRORetrysDelete202Retry200Operation");
+            IOperation nextLinkOperation = NextLinkOperationImplementation.Create(pipeline, request.Method, request.Uri.ToUri(), response, OperationFinalStateVia.Location);
+            _operation = new OperationInternal(clientDiagnostics, nextLinkOperation, response, "LRORetrysDelete202Retry200Operation");
         }
 
         /// <inheritdoc />
-        public override string Id => _operation.Id;
+#pragma warning disable CA1822
+        public override string Id => throw new NotImplementedException();
```

Comment:
Why not throw from `OperationInternal`? If we throw here, after implementing the rehydration, we need to uptake again?

Reply:
because if you look at the `OperationInternals` which is the previous implementation we use, you'll find this is the exact codes. This PR replaces `OperationInternals` with `OperationInternal`, basically unfold the codes of `OperationInternals` here.

The previous relationship is `{RP_Action}Operation` --> `OperationInternals` --> `OperationInternal`.
Now the relationship is `{RP_Action}Operation` --> `OperationInternal`. Some codes in `OperationInternals` are not generated in `{RP_Action}Operation`.

Reply:
I mean, why not add this line to `OperationInternal`
```C#
public override string Id => throw new NotImplementedException();
```
And leave the code in XXXOperation as 
```C#
public override string Id => _operation.Id;
```